CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Changed
+- The version of Conjur OSS used by the Marketplace app is bumped to 1.6.0.
+- The CA certificate Common Name (CN) can now be manually configured as an environment variable for development testing via the build.sh script.
```

Comment:
this line doesn't belong in the CHANGELOG, which should be used to describe user-facing changes to the project. is there a user impact of this change?